### PR TITLE
Ovl_Bg_Spot01_Idomizu OK

### PR DESCRIPTION
**Pull request by tom-overton**

### CODE REVIEW

File: src/overlays/actors/ovl_Bg_Spot01_Idomizu/z_bg_spot01_idomizu.c [hunk 1]
```diff
@@ -29,13 +28,66 @@ const ActorInit Bg_Spot01_Idomizu_InitVars = {
...
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot01_Idomizu/func_808ABB84.s")
+    gSPMatrix(gfxCtx->polyXlu.p++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_bg_spot01_idomizu.c", 232),
+              G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot01_Idomizu/BgSpot01Idomizu_Update.s")
+    framesTemp = globalCtx->state.frames;
+    yFramesComponent = framesTemp;
+    xFramesComponent = yFramesComponent % 128;
+    gSPSegment(gfxCtx->polyXlu.p++, 0x08,
+               Gfx_TwoTexScroll(globalCtx->state.gfxCtx, 0, 127 - xFramesComponent, yFramesComponent & 0x7F, 32, 32, 1,
```
Comment (ethteck) [inline src/overlays/actors/ovl_Bg_Spot01_Idomizu/z_bg_spot01_idomizu.c:88]: Can the `& 0x7F`s here become `% 128`?

Comment (tom-overton) [reply src/overlays/actors/ovl_Bg_Spot01_Idomizu/z_bg_spot01_idomizu.c:88]: Unfortunately not. It doesn't match with `% 128`

Comment (tom-overton) [reply src/overlays/actors/ovl_Bg_Spot01_Idomizu/z_bg_spot01_idomizu.c:88]: To be specific, the diff becomes this when you do `% 128` here

image

Comment (ethteck) [reply src/overlays/actors/ovl_Bg_Spot01_Idomizu/z_bg_spot01_idomizu.c:88]: Weird, okay. Well that may explain why I had such trouble matching this :P

**Pull request merged**